app/src/main/java/de/tum/in/tumcampusapp/component/tumui/calendar/model/Event.kt

```diff
@@ -37,7 +38,7 @@ data class Event(
         return CalendarItem(
                 id ?: "", status ?: "", url ?: "", title,
                 description ?: "", startTime ?: DateTime(),
-                endTime ?: DateTime(), location ?: "", false
+                endTime ?: DateTime(), Utils.stripHtml(location ?: ""), false
```

Comment:
Shouldn't we rather decode them instead of removing them? They might be adding context to the text.

See: https://stackoverflow.com/questions/2918920/decode-html-entities-in-android

Reply:
This is essentially what it does. `Utils.stripHtml` is a wrapper around `Html.fromHtml().toString()` so it converts the HTML to the actual string representation. I understand the confusion because of the function name though. Would it maybe be better to rename the `stripHtml` function since that is a wrong naming?

Reply:
Ah right, I didn't check. Its fine then. I might adapt the naming or if you really want, feel free to open up a second PR.